src/contentsharecontroller/ContentShareMediaStreamBroker.ts

```diff
@@ -23,6 +23,7 @@ export default class ContentShareMediaStreamBroker implements MediaStreamBroker
 
   async acquireAudioInputStream(): Promise<MediaStream> {
     if (this._mediaStream.getAudioTracks().length === 0) {
+      this.logger.info('Synthesizing fake audio stream');
```

Comment:
Should we not use the word fake? Maybe something like 'No audio stream available. Synthesize an audio stream'

Reply:
Changed!!